doc/apiv3-documentation.apib

```diff
@@ -2584,6 +2584,15 @@ all properties of the linking work package, including properties added by plug-i
 they can occur as properties or as linked properties. A client has to consult the schema to resolve
 the human readable name of custom fields.*
 
+*Properties that cannot be set directly on parent work packages are inferred from their children instead:*
+
+* `start date` is the earliest start date from its children
+* `due date` is the latest due date from its children
+* `estimated time` is the sum of estimated times from its children
+* `percentage done` is the average of the sum of its children percentages done
```

Comment:
you should format those names as `camelCase`, thus refer to their actual identifiers...